src/test/pmem2_badblock_mocks/pmem2_badblock_mocks.h

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright 2019-2020, Intel Corporation */
+
+/*
+ * pmem2_badblock_mocks.h -- definitions for pmem2_badblock_mocks test
+ */
+
+#include <ndctl/libndctl.h>
+#include "extent.h"
+
+/*
+ *        fd ==  99 - regular file, did not found any matching device
```

Comment:
It took me a while to understand this. I think it might be easier if you make namespaces and regions into real bitmasks.

Reply:
Done